posthog/cdp/templates/discord/template_discord.py

```diff
@@ -0,0 +1,62 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate, HogFunctionSubTemplate, SUB_TEMPLATE_COMMON
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="beta",
```

Comment:
I wonder if we want this and Teams to be `status="free"` like the Slack one (perhaps a discussion for standup?)